consensus/ibft/sign.go

```diff
@@ -184,35 +172,75 @@ func verifyCommittedFields(
 		return fmt.Errorf("empty committed seals")
```

Comment:
we have const for this err: ErrEmptyCommittedSeals

Reply:
Resolved in the latest PR